daedalus/package.json

```diff
@@ -51,6 +51,7 @@
     "blakejs": "^1.1.0",
     "bs58": "^4.0.1",
     "json-loader": "^0.5.7",
+    "purescript": "^0.11.5",
```

Comment:
Are you sure this is needed?

Reply:
@adinapoli-iohk good catch! Thanks, I will remove it.

That thing is added by `npm install` on my machine automatically and usually I do check it - but it was late. Thanks

added to TODO list